content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
-                <div class="item">
-                  <div class="content">
-                    <div class="header"><i>All the features from the Base plan</i></div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Ad-free user</div>
-                    <div class="description">stop seeing ads on any Read the Docs hosted documentation when logged in to your account</div>
-                  </div>
-                </div>
-                <div class="item">
-                  <div class="content">
-                    <div class="header">Ad-free documentation</div>
-                    <div class="description">stop showing ads to readers on hosted documentation</div>
-                  </div>
-                </div>
-              </div>
-
-            </div>
-          </div>
         </div>
-        <!-- end card -->
-
       </div>
 
-
-      <div class="ui segment basic vertical padded center aligned">
-        <p>
-          <a class="ui button blue tiny"  data-analytics="community-signup" href="https://readthedocs.org/accounts/signup">Get started for free</a>
-        </p>
+      <div class="ui basic center aligned padded segment">
+        <a class="ui teal button" href="https://readthedocs.org/accounts/signup" data-analytics="community-signup">Sign up for an account</a>
       </div>
 
     </div>
-  </section>
-
-
-  <!-- Intro -->
-  <section class="ui vertical segment padded" id="intro">
-    <div class="ui container">
-
-      <div class="ui vertical segment basic padded">
-
-        <div class="ui grid center aligned">
-          <div class="column ten wide computer sixteen wide mobile">
-
-            <h3 class="ui header massive"><span class="ui header">About our platforms</span></h3>
-            <p>
-              A question our users often have is what the difference is between Read the Docs Community and Read the Docs for Business. The features available on both platforms are the same. The primary difference is the audience and use cases that are supported.
-            </p>
-
-          </div>
-        </div>
-
-      </div>
-
-      <div class="ui vertical segment basic padded center aligned">
-        <a href="https://docs.readthedocs.io/en/stable/choosing-a-site.html#choosing-between-our-two-sites" class="ui button black huge">Choosing between our two sites</a>
-      </div>
-
+  </div>
+</section>
+
+{# Compare plan features with a table #}
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call pricing.feature_table() %}
+      {{ pricing.feature_row("Team management", "Control organization access with team permissions", "fa-users", on_community=False) }}
+      {{ pricing.feature_row("Email support", "Get help with your projects and documentation", "fa-message-question", None, "2 business days", "2 business days", "1 business day") }}
```

Comment:
We do actually offer pretty good email support on community, but probably better to underpromise here.  🤷

Reply:
Yeah, i was not entirely sure how to frame this one in particular. We support the community platform of course, but support isn't necessarily part of the community product. It is definitely part of the commercial product though.

I thought about just making this one a check mark, and the commercial plans would state the "2 business days"/etc in this row. It's ambiguous what community support is in this case, but at least looks like we're responsible for something :smile: 

![image](https://user-images.githubusercontent.com/1140183/201777918-17db19d3-36b9-44f1-90d3-13e675c4ed34.png)

Reply:
I actually do like the check, it's a rather simple solution. If we get questions here we can elaborate, but I actually think the check works well for this purpose, and differentiates commercial nicely.